adminapi/cli.py

```diff
@@ -107,7 +107,7 @@ def print_server(server, attribute_ids):
             continue
 
         value = server[attribute_id]
-        if value in [None, True, False]:
+        if value is None:
```

Comment:
Well spotted.  I din't knew `0 == False and 1 == True` on Python.  This fix would break the representation for booleans.  `if any(value is v for v in (None, True, False)` does the correct thing.